src/screens/stationSelect.tsx

```diff
@@ -85,7 +85,11 @@ const StationSelectScreen: React.FC<StationSelectProps> = () => {
         mode="contained"
         onPress={() => {
           if (stationSelectionIsValid(departStation, arriveStation)) {
-            openUrlInBrowser(departStation, arriveStation);
+            // openUrlInBrowser(departStation, arriveStation);
```

Comment:
might as well remove this rather than commenting it out, you can always get it back from git history if you need it